concepts/evaluation/queue.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Annotation Queues"
+---
+
+The Literal AI platform helps you organise and smoothen the human evaluation process by easily setting-up annotation queues.
+Each queue let you add data for review and provides an accessible and efficient ui for domain experts to score, tag and add to datasets crucial insights.
+
+## Creating an annotation queue
+
+You can access your queues on the "Annotation Queues" page, and create them by pressing the "+" cta in the top right corner of the table.
+
+<img src="/images/create-annotation-queue.png" alt="Create annotation queue image" />
```

Comment:
would you consider using [mintlify's frames](https://github.com/Chainlit/literal-docs/blob/main/concepts/observability/multimodal.mdx?plain=1#L21) ? they look good and allow you to add a caption to the image

Reply:
Looks cool didn't know about those.